### DISCUSSION TITLE
Parentheses use for inferred fusions

### DISCUSSION THREAD

**Discussion by ahwagner** [cancervariants/fusions#30] [Polls]
## Background
Our current draft recommendations specify the [use of parentheses surrounding a double-colon for describing a gene fusion that is called by inference](https://fusions.cancervariants.org/en/latest/nomenclature.html#inferred-fusions), e.g. through detection of a rearrangement consistent with the fusion. However, the absence of these parentheses can alternately mean "directly detected" or (as is current practice) "not specified", with no way to discern between the two. A similar issue was raised regarding direct vs. inferential description of RNA and protein variants, which is currently [under active discussion in HGVS](https://github.com/HGVSnomenclature/hgvs-nomenclature/discussions/18). This discussion may resolve in the removal of internal parentheses from HGVS r. and p. strings.

To avoid a similar outcome in the fusion nomenclature, we should either A) step back our recommended nomenclature to not include the parentheses and leave descriptions to text only, or B) add a new set of symbols (e.g. `{ }`, `[ ]` or `< >`) in addition to parentheses, with the specific meaning that the fusion was directly observed.

## Option A: remove parenthetical notation for inferred fusions
Under Option A, an inferred gene fusion would be described in plain text:

> A t(9;22)(q34;q11) reciprocal translocation resulting in an inferred _BCR_::_ABL1_ fusion.

Where a directly measured gene fusion would similarly be described in plain text:

> A t(9;22)(q34;q11) reciprocal translocation resulting in a _BCR_::_ABL1_ fusion and confirmed by RT-PCR.

## Option B: add alternate convention for directly measured fusions
Under Option B, an inferred gene fusion would be described using parentheses:

> A t(9;22)(q34;q11) reciprocal translocation resulting in a _BCR_(::)_ABL1_ fusion.

Where a directly measured gene fusion would similarly be described using an alternate symbol (e.g. `[ ]`):

> A t(9;22)(q34;q11) reciprocal translocation resulting in a _BCR_[::]_ABL1_ fusion.

## Request
Please indicate your preference for notating inferred fusions in the below poll, and add any supporting rationale or comments to this discussion thread.

**Comment by ahwagner** [comment]
My personal preference is to remove the parenthetical notation (Option A), as this provides us the flexibility of reintroducing it or an alternate convention at a later date once the current conversation around parentheses in HGVS is resolved.

**Comment by Laveniya** [comment]
I agree, Alex. I also agree with Gordana’s points in yesterday’s meeting - nomenclature is used interchangeably, and sometimes inappropriately. 

A dumb question - by option A, does this mean that all fusions, whether inferred or directly measured, require a plain text description? 

A suggestion - it would be great to perhaps give examples of plain text descriptions in the manuscript for ‘inferred fusion assays’, e.g. FISH - examples may increase uptake of the recommendations. Happy to help with this if needed.

**Reply by ahwagner** [reply]
> A dumb question - by option A, does this mean that all fusions, whether inferred or directly measured, require a plain text description?

No, I don't think it would require that in all cases, though we will certainly recommend this information is shared in some form. It depends on how the fusion is being described; a clinical report might have a flag applied, or autogenerated text similar to this, in cases where the fusion was not confirmed by direct observation. A database record would still maintain the information in a computable format following our [information model](https://fusions.cancervariants.org/en/latest/information_model.html#assayed-elements), and would probably display inferred/observed as a field in table, for example. Other resources might choose to wrap the whole fusion in parentheses to indicate it as a consequence (either predicted or directly observed), e.g.:

> t(9;22)(q34;q11) (_BCR_::_ABL1_)

Lots of possibilities, I don't think we have had any discussion about recommended ways to represent this information beyond as parentheses in the nomenclature.

**Reply by Laveniya** [reply]
Thanks Alex, that makes sense, thanks for answering my question!
